dev/com.ibm.ws.install.featureUtility/src/com/ibm/ws/install/featureUtility/props/FeatureUtilityProperties.java

```diff
@@ -153,8 +162,26 @@ private static boolean parseProperties(Properties properties) {
                     }
                     String repoName = split[0];
                     String option = split[split.length - 1]; // incase there are periods in the key, cant use [1]
+
                     if(repoMap.containsKey(repoName)){
-                        repoMap.get(repoName).put(option, value.toString());
+			if (option.compareToIgnoreCase("password") == 0) {
+			    String decodedPwd = value.toString();
+			    try {
+				decodedPwd = PasswordUtil.decode(value.toString());
+			    } catch (InvalidPasswordDecodingException ipde) {
+				logger.log(Level.FINE,
```

Comment:
Invalid password should follow same pattern as other places? to set decodedPwd back to the plain text from `value.toString()`